src/compiler/emitter.ts

```diff
@@ -455,6 +455,7 @@ namespace ts {
                 inlineSourceMap: compilerOptions.inlineSourceMap,
                 extendedDiagnostics: compilerOptions.extendedDiagnostics,
                 onlyPrintJsDocStyle: true,
+                omitBraceSourcePositions: true,

```

Comment:
Can we make this name more obviously about source _maps_? I think most people dealing with the emitter API have .js/.d.ts emit in their minds, where this name makes no sense.

Reply:
I can change the name, though this property is marked `/*@internal*/` so it shouldn't impact API consumers outside of our own team.